app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
The `people` dictionary is used to store `Person` instances by name. This could lead to issues if names are not unique, as it would overwrite existing entries. Consider using a unique identifier for each person.